<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 250, 254)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_only_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(31, 149, 239);	
}
QPushButton{
	color: white;
	height: 30px;
	border: none;
	border-radius: 10px;
}
QPushButton:checked{
	background-color: #F5FAFE;
	color: #1F95EF;
	font-weight: bold;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../../../Images/SGBD/res.qrc">:/images/Capture d’écran du 2024-04-10 00-58-39.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="PV_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../../Images/SGBD/res.qrc">
             <normaloff>:/images/Capture d’écran du 2024-04-11 14-50-36.png</normaloff>:/images/Capture d’écran du 2024-04-11 14-50-36.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SEM_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../../Images/SGBD/res.qrc">
             <normaloff>:/images/Capture d’écran du 2024-04-11 14-48-16.png</normaloff>:/images/Capture d’écran du 2024-04-11 14-48-16.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>616</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../../Images/SGBD/res.qrc">
           <normaloff>:/images/Capture d’écran du 2024-04-10 01-45-32.png</normaloff>:/images/Capture d’écran du 2024-04-10 01-45-32.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="icon_name_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(31, 149, 239);	
	color:white;
}
QPushButton{
	color: white;
	text-align:left;
	height: 30px;
	border:none;
	padding-left: 10px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
}
QPushButton:checked{
	background-color: #F5FAFE;
	color: #1F95EF;
	font-weight: bold;
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../../../Images/SGBD/res.qrc">:/images/Capture d’écran du 2024-04-10 00-58-39.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Directeur_Etude</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="PV_2">
           <property name="text">
            <string>PV</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../../Images/SGBD/res.qrc">
             <normaloff>:/images/Capture d’écran du 2024-04-11 14-50-36.png</normaloff>:/images/Capture d’écran du 2024-04-11 14-50-36.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SEM_2">
           <property name="text">
            <string>Rapport Semestre</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../../Images/SGBD/res.qrc">
             <normaloff>:/images/Capture d’écran du 2024-04-11 14-48-16.png</normaloff>:/images/Capture d’écran du 2024-04-11 14-48-16.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>616</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Deconnexion</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../../Images/SGBD/res.qrc">
           <normaloff>:/images/Capture d’écran du 2024-04-10 01-45-32.png</normaloff>:/images/Capture d’écran du 2024-04-10 01-45-32.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="main_menu" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="header_widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Images/SGBD/res.qrc">
              <normaloff>:/images/Capture d’écran du 2024-04-10 01-58-39.png</normaloff>:/images/Capture d’écran du 2024-04-10 01-58-39.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>218</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_10">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../../../Images/SGBD/res.qrc">
                <normaloff>:/images/Capture d’écran du 2024-04-10 10-06-07.png</normaloff>:/images/Capture d’écran du 2024-04-10 10-06-07.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>218</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_11">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Images/SGBD/res.qrc">
              <normaloff>:/images/Capture d’écran du 2024-04-10 10-10-46.png</normaloff>:/images/Capture d’écran du 2024-04-10 10-10-46.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_PV">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>250</y>
             <width>171</width>
             <height>121</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>aakar</family>
             <pointsize>50</pointsize>
            </font>
           </property>
           <property name="text">
            <string>P.V</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_Sem">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>300</y>
             <width>511</width>
             <height>121</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>aakar</family>
             <pointsize>50</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Rapport Semestre</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../../../Images/SGBD/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_9</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_name_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SEM_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>SEM_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PV_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>PV_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PV_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>PV_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SEM_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>SEM_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>842</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>900</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>835</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>892</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
